addons/spreadsheet/static/src/pivot/odoo_pivot_loader.js

```diff
@@ -77,7 +94,15 @@ export class OdooPivotLoader {
      * @returns {Promise<OdooFields>} Fields of the model
      */
     async getFields(model) {
-        return await this.odooDataProvider.serverData.fetch(model, "fields_get");
+        try {
+            return await this.odooDataProvider.serverData.fetch(model, "fields_get");
+        } catch (e) {
+            if (e instanceof RPCError && e.code === 404) {
+                // The model does not exist.
+                throw new ModelNotFoundError(model);
```

Comment:
nitpick: comment seems a bit redundant

From my 20 seconds searching in the docs I couldn'tfind the references to RPC error codes, I'll trsut you on the 404 for model not found :) But Maybe there's other interesting error code to catch ?